send_message_to_msteams.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import random
+from abc import ABC, abstractmethod
+
+import pymsteams
+import requests
+from requests.exceptions import ConnectionError, HTTPError
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
+# Constants
+WIDTH = random.randint(300, 800)
+HEIGHT = random.randint(300, 800)
+MESSAGE_COLOR = "#F8C471"
+TEAMS_WEBHOOK_URL = "your_webhook"  # TODO: store credentials in a secure way
+
+class ContentGenerator(ABC):
+    @abstractmethod
+    def get_content(self):
+        """ Method create connection to API and select random image/quote/etc."""
+        pass
+
+
+class QuoteGenerator(ContentGenerator):
```

Comment:
Probably those names should be more specific since they fetch an info from an exact source